source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -324,6 +303,11 @@ void BioXASAppController::setupUserInterface()
 	addPersistentView(new BioXASPersistentView());
 }
 
+void BioXASAppController::setupBeamlineSignalConnections()
```

Comment:
Do all CLSBeamline subclasses have to implement `setupBeamlineSignalConnections()`? I can see it being useful for some/most, but it doesn't appear to be used here.